server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -79,9 +79,9 @@ export function getCreationToken(
 	// Current time in seconds
 	const tokenClaims = decode(token) as ITokenClaims;
 
-	const { tenantId, user } = tokenClaims;
+	const { tenantId, user, jti } = tokenClaims;
 
-	return generateToken(tenantId, documentId, key, [], user, lifetime);
+	return generateToken(tenantId, documentId, key, [], user, jti /* JwtId */, lifetime);
```

Comment:
nit: no need for the comment, the `jti` variable name is self explanatory and also already matches the param name